src/core/common/time.hpp

```diff
@@ -64,19 +64,13 @@ class Time
      */
     static const uint32_t kMaxDuration = ~static_cast<uint32_t>(0UL);
 
-    /**
-     * This is the default constructor for a `Time` object.
-     *
-     */
-    Time(void) {}
-
     /**
      * This constructor initializes a `Time` object with a given value.
      *
      * @param[in] aValue   The numeric time value to initialize the `Time` object.
      *
      */
-    explicit Time(uint32_t aValue) { SetValue(aValue); }
+    explicit Time(uint32_t aValue = 0) { SetValue(aValue); }
```

Comment:
I think `Time` is indeed POD. But if the compiler can't understand it, maybe we should ignore this specific warning?

Reply:
okay, suppress the warning also works for me. updated.